### PR TITLE
QA-530: ci: Use pre-built Docker images

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Comment by lluiscampos** [comment]
Depends on:
* https://github.com/mendersoftware/mender-test-containers/pull/302

File: docker-build-package [hunk 1]
```diff
@@ -143,9 +143,11 @@ echo
 docker run --rm \
         --volume $(pwd)/recipes:/recipes \
         --volume $(pwd)/${output_dir}:/output \
+        --volume $(pwd)/mender-deb-package:/script \
         -e MENDER_PRIVATE_REPO_ACCESS_TOKEN="${MENDER_PRIVATE_REPO_ACCESS_TOKEN}" \
         -e MENDER_PRIVATE_REPO_ACCESS_USER="${MENDER_PRIVATE_REPO_ACCESS_USER}" \
         ${IMAGE_NAME_PREFIX}-${DISTRO}-${RELEASE}-${ARCH} \
+        /script \
```
Comment (merlin-northern) [inline docker-build-package:150]: I would change that name to something else, as much as I love scripts :) I also love other cool things.

File: docker-build-package [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 set -e
 
-IMAGE_NAME_PREFIX=mendersoftware/mender-dist-packages:builder
+IMAGE_NAME_PREFIX=mendersoftware/mender-test-containers:mender-dist-packages-builder
```
Comment (merlin-northern) [inline docker-build-package:19]: I understand that there is a reason that we are using mender-test-containers with a tag, and not a dedicated image?

**Review by merlin-northern — APPROVED** [review]
looks good, left some comments. one general comment: could we have a global check, before the build phase, that `${IMAGE_NAME_PREFIX}-${DISTRO}-${RELEASE}-${ARCH}` image actually exists?

File: docker-build-package (hunk 1 above)
Comment (lluiscampos) [reply docker-build-package:150]: Like what? Please suggest something specific.

File: docker-build-package (hunk 2 above)
Comment (lluiscampos) [reply docker-build-package:19]: The reason is to be consistent with every other image built in `mender-test-containers` repository, as listed [here](https://github.com/mendersoftware/mender-test-containers/#mender-test-containers).

File: docker-build-package (hunk 1 above)
Comment (merlin-northern) [reply docker-build-package:150]: > Like what? Please suggest something specific.

you know naming. let's do it like this: if you are ok with this name, lets keep it.

**Comment by lluiscampos** [comment]
This is the counterpart (and some clean-ups) of https://github.com/mendersoftware/mender-test-containers/pull/302

@mzedel @oldgiova @merlin-northern Could you guys take a look now?

**Review by oldgiova — APPROVED** [review]

**Pull request merged**